index.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio-website</title>
+  </head>
+  <body>
+    <header>
+      <img src=".png\geometry_1.png" />
+      <button class="menu">
```

Comment:
Please notice that this image shouldn't be an `img` element. Instead, it should be the [background image](https://www.w3schools.com/cssref/pr_background-image.php) of your container. This will make your code semantically accurate and give you an easier time with styling 😃. You can [check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more information.